components/tag/src/tag-list/tag-list.ts

```diff
@@ -196,6 +200,17 @@ export class DtTagList implements AfterContentInit, OnDestroy {
     });
     this._tagAddSubscriptions = [];
   }
+
+  /**
+   * @internal evaluates whether to display the x more button
+   */
+  _toDisplayMoreButton(): void {
+    if (!this._isOneLine && !this._showAllTags && this._hiddenTagCount > 0) {
```

Comment:
according to our style guide please use a conditional operator for that and furthermore use the function directly in the template no need to store in a variable


```typescript
_showDisplayMoreButton(): boolean {
  return !this._isOneLine && !this._showAllTags && this._hiddenTagCount > 0
}
```